versioned_docs/version-24.04/administration/centreon-ha/acceptance-guide.md

```diff
@@ -0,0 +1,712 @@
+
+### Perform the test
+
+We're assuming that node 1 is the active node and node 2 is the passive node ([check the state of the cluster](#check-the-state-of-the-cluster) if you need to).
+
+To perform this test, run the `iptables` commands on the **active central node**:
+
+```bash
+iptables -A INPUT -s @CENTRAL_NODE1_IPADDR@ -j DROP
+iptables -A OUTPUT -d @CENTRAL_NODE1_IPADDR@ -j DROP
+iptables -A INPUT -s @DATABASE_NODE1_IPADDR@ -j DROP
+iptables -A OUTPUT -d @DATABASE_NODE1_IPADDR@ -j DROP
+iptables -A INPUT -s @DATABASE_NODE2_IPADDR@ -j DROP
+iptables -A OUTPUT -d @DATABASE_NODE2_IPADDR@ -j DROP
+iptables -A INPUT -s @QDEVICE_IPADDR@ -j DROP
+iptables -A OUTPUT -d @QDEVICE_IPADDR@ -j DROP
+```
+
+The passive central node is now excluded from the cluster.
+
+If you run `pcs status` on the active central node:
+
+* The resources and the cluster are still working.
+* The passive central node is seen `offline` on the active node:
+
+```text
+Cluster name: centreon_cluster
+Stack: corosync
+Current DC: @CENTRAL_MASTER_NAME@ (version 1.1.23-1.el8_9.1-9acf116022) - partition with quorum
+Last updated: Thu May  5 10:34:05 2022
+Last change: Thu May  5 09:09:50 2022 by root via crm_resource on @CENTRAL_MASTER_NAME@
+
+4 nodes configured
+21 resource instances configured
+
+Online: [ @DATABASE_MASTER_NAME@ @CENTRAL_MASTER_NAME@ @DATABASE_SLAVE_NAME@ ]
+OFFLINE: [ @CENTRAL_SLAVE_NAME@ ]
+
+Full list of resources:
+
+ Master/Slave Set: ms_mysql-clone [ms_mysql]
+     Masters: [ @DATABASE_MASTER_NAME@ ]
+     Slaves: [ @DATABASE_SLAVE_NAME@ ]
+     Stopped: [ @CENTRAL_MASTER_NAME@ @CENTRAL_SLAVE_NAME@ ]
+ vip_mysql      (ocf::heartbeat:IPaddr2):       Started @DATABASE_MASTER_NAME@
+ Clone Set: php-clone [php]
+     Started: [ @CENTRAL_MASTER_NAME@ ]
+     Stopped: [ @DATABASE_MASTER_NAME@ @DATABASE_SLAVE_NAME@ @CENTRAL_SLAVE_NAME@ ]
+ Clone Set: cbd_rrd-clone [cbd_rrd]
+     Started: [ @CENTRAL_MASTER_NAME@ ]
+     Stopped: [ @DATABASE_MASTER_NAME@ @DATABASE_SLAVE_NAME@ @CENTRAL_SLAVE_NAME@ ]
+ Resource Group: centreon
+     vip        (ocf::heartbeat:IPaddr2):       Started @CENTRAL_MASTER_NAME@
+     http       (systemd:httpd24-httpd):        Started @CENTRAL_MASTER_NAME@
+     gorgone    (systemd:gorgoned):     Started @CENTRAL_MASTER_NAME@
+     centreon_central_sync      (systemd:centreon-central-sync):        Started @CENTRAL_MASTER_NAME@
+     cbd_central_broker (systemd:cbd-sql):      Started @CENTRAL_MASTER_NAME@
+     centengine (systemd:centengine):   Started @CENTRAL_MASTER_NAME@
+     centreontrapd      (systemd:centreontrapd):        Started @CENTRAL_MASTER_NAME@
+     snmptrapd  (systemd:snmptrapd):    Started @CENTRAL_MASTER_NAME@
+
+Daemon Status:
+  corosync: active/enabled
+  pacemaker: active/enabled
+  pcsd: active/enabled
+```
+
+If you run `pcs status` on the passive node:
+
+* All resources appear stopped on the passive node
```

Comment:
Explain that this is because of "partition WITHOUT quorum" (as seen in the output from pcs status). No quorum, so pacemaker stops the resources.